src/main/scala/firrtl/Emitter.scala

```diff
@@ -372,8 +380,9 @@ class VerilogEmitter extends Emitter {
         initials += Seq("`endif")
       }
 
-      def instantiate(n: String,m: String, es: Seq[Expression]) {
-         instdeclares += Seq(m, " ", n, " (")
+      def instantiate(n: String, m: String, es: Seq[Expression], params: Seq[Param]): Unit = {
+         val ps = if (params.nonEmpty) params map stringify mkString ("#(", ", ", ") ") else ""
+         instdeclares += Seq(m, " ", ps, n ," (")
          es.zipWithIndex foreach {case (e, i) =>
```

Comment:
I wasn't really rewriting the whole function although I can fix this adjacent line